hw_7.class.py

```diff
@@ -0,0 +1,154 @@
+# Задача-1
+# У вас есть список(list) IP адрессов. Вам необходимо создать
+# класс который будет иметь методы:
+# 1) Получить список IP адресов
+# 2) Получить список IP адресов в развернутом виде
+# (10.11.12.13 -> 13.12.11.10)
+# 3) Получить список IP адресов без первых октетов
+# (10.11.12.13 -> 11.12.13)
+# 4) Получить список последних октетов IP адресов
+# (10.11.12.13 -> 13)
+class Ip:
+    def __init__(self, ip):
+        self._ip = ip
+
+    def get_ip(self):
+        return self._ip
+
+    def get_ip_expand(self):
+        for item in self._ip:
+            self.ip_expand = item.split('.')
+            self.ip_expand = '.'.join(self.ip_expand[::-1])
+            print(f'{self.ip_expand}')
+
+    def get_ip_without_first(self):
+        for item in self._ip:
+            self.ip_without_first = item[3:]
+            print(self.ip_without_first)
+
+    def get_last_ip(self):
+        for item in self._ip:
+            self.last_ip = item[-2:]
+            print(self.last_ip)
+
+d=Ip(['12.13.15.14', '10.11.12.13'])
+d.get_ip()
+d.get_ip_expand()
+d.get_ip_without_first()
+d.get_last_ip()
+
+# Задача-2
+# У вас несколько JSON файлов. В каждом из этих файлов есть
+# произвольная структура данных. Вам необходимо написать
+# класс который будет описывать работу с этими файлами, а
+# именно:
+# 1) Запись в файл
+# 2) Чтение из файла
+# 3) Объединение данных из файлов в новый файл
+# 4) Получить путь относительный путь к файлу
+# 5) Получить абсолютный путь к файлу
+#
+def get_file_tools(my_file):
+    """ Function creates an instance of the FileTools class.
+    Actions: reading and writing to a file, merging files
+    getting relative and absolute paths"""
+    import os.path
+
+    if os.path.exists(my_file):
+
+        class FileTools:
+
+            def __init__(self, file_1):
+                self._file_1 = file_1
+
+            def write_this(self, text):
+                with open(self._file_1, 'w') as writer:
+                    writer.write(text)
+
+            def read_it(self):
+                with open(self._file_1, 'r') as reader:
+                    for line in reader:
+                        print(line, end='')
+
```

Comment:
It would be nice to add one more method for JSON validation to catch JSONDecodeError